src/app.ts

```diff
@@ -20,7 +20,7 @@ import { applicationErrorMapper, httpErrorHandler, unknownErrorHandler } from '.
 
 const configVersionSupported: ConfigVersion = 1;
 
-export function generateServerlessRouter(fhirConfig: FhirConfig, supportedGenericResources: string[]) {
+export function generateServerlessRouter(fhirConfig: FhirConfig, supportedGenericResources: string[]): any {

```

Comment:
Can you check if the type here is supposed to be `Express`?

And then at the top
`import express, { Express } from 'express';`